cfe_internal/enterprise/CFE_hub_specific.cf

```diff
@@ -765,27 +765,47 @@ bundle agent cfe_internal_enterprise_policy_analyzer
 
       "policy_analyzer_enabled" expression => fileexists( $(analyzer_flagfile) );
 
-    files:
+  files:
 
-    policy_analyzer_enabled::
+    enterprise_edition.policy_server::
 
-      "$(analyzer_dir)/."
+      "$(cfe_internal_hub_vars.docroot)/analyzer/."
```

Comment:
Why the change here and not on line 785 below?

Reply:
Yeah, it got me for a few minutes as well.

Because analyzer_dir != docroot/analyzer.  Analyzer_dir is in opt, this docroot analyzer path is a directory where the web server writes the enablement flag file if someone clicks that button in MP. Then analyzer_dir is where the MPF is copied and where the webserver reads the policy from for rendering in MP.

Reply:
So this is really a bug-fix of sorts. Ack.

Reply:
MMMmm, I don't think so, I think that previously the web server had the ability to create the directory in the docroot if it didn't exist.